examples/flax/image-captioning/run_image_captioning_flax.py

```diff
@@ -551,11 +551,10 @@ def tokenization_fn(examples, max_target_length):
         targets = captions
 
         model_inputs = {}
-        # Setup the tokenizer for targets
-        with tokenizer.as_target_tokenizer():
-            labels = tokenizer(
-                targets, max_length=max_target_length, padding="max_length", truncation=True, return_tensors="np"
-            )
+
+        labels = tokenizer(
+            targets, max_length=max_target_length, padding="max_length", truncation=True, return_tensors="np"
```

Comment:
Perhaps should be an explicit `kwarg` here - not sure if it makes a difference.  

```suggestion
            text_target=targets, max_length=max_target_length, padding="max_length", truncation=True, return_tensors="np"
```

Reply:
This one was actually a mistake from the user: the tokenizer is a GPT2-like which does not have a target mode (it won't bug but it's the same thing). It doesn't really matter which one we use, so since it's labels, let's use the target keyword.